docs/source/multi_gpu.rst

```diff
@@ -207,6 +207,16 @@ Note in particular the difference between `gpus=0`, `gpus=[0]` and `gpus="0"`.
     occupied by other processes. This is especially useful when GPUs are configured
     to be in "exclusive mode", such that only one process at a time can access them.
 
+.. testcode::
```

Comment:
shouldn't this move up to the other examples? or perhaps the note should become a regular text so that it connects to this new example  here